lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -56,6 +60,17 @@ def initialize
         @target_command_names = ServerVerifier::Options.new.target_command_names
 
         @read_timeout = Groonga::Client::Default::READ_TIMEOUT
+
+        @mail_notification = false
+        @mail_subject_on_succeeded = "Succeeded"
+        @mail_subject_on_failure = "Failed"
+        @mail_from = "groonga-query-log@example.com"
+        @mail_to = "noreply@localhost"
+        @smtp_server = "localhost"
+        @smtp_auth_user = "unknown"
+        @smtp_auth_password = "unknown"
+
+        @start_time = Time.now
```

Comment:
It should execute just before `tester.run`. If we execute this here, we count needless time.